week-2/Homework/mandatory/4-sorting-algorithm.js

```diff
@@ -21,9 +21,36 @@ You don't have to worry about making this algorithm work fast! The idea is to ge
 */
 //1)Create the function inside, Think about nested loops
 function sortAges(arr) {
-  //create function here
+  let arr1 = [];
+  for (let i = 0; i < arr.length; i++) {
+    if (Number.isInteger(arr[i])) {
+      arr1.push(arr[i]);
+    }
+  }
+  let arr2 = [];

```

Comment:
arr2, is not used later on, so a good option is to declare it inside the loop where it's being used, and not as an array (you are using it as a number variable)